changelog.md

```diff
@@ -1,2 +1,13 @@
+#1.5.1
+Upgrade karma to 2.0.5 to resolve a security vulnerability
+See: https://nvd.nist.gov/vuln/detail/CVE-2017-16113
+
+#1.5
+Added support for NativeScript 3.4
```

Comment:
Strange that this is marked as new

Reply:
Hm, yes, it is weird, this file was not changed at all.